src/pages/auth/chat/room/ChatRoom.tsx

```diff
@@ -4,13 +4,28 @@ import * as S from "./style";
 import Screen from "./Screen";
 import ExitBtn from "./ExitBtn";
 import SendBtn from "./SendBtn";
-
+import { getSocket } from "socket/socket";
+import { useEffect } from "react";
```

Comment:
```suggestion
import { useEffect } from "react";

```

import문 이후에 한 줄 추가해주시면 가독성에 도움이 될것 같습니다 ㅎㅎ